tests/servers/test_liveserverthread.py

```diff
@@ -0,0 +1,33 @@
+from unittest import SkipTest
+
+from django.db import DEFAULT_DB_ALIAS, connection, connections
+from django.test import LiveServerTestCase, TestCase
+from django.test.testcases import is_connection_sqlite_in_memory
+
+
+class LiveServerThreadTest(TestCase):
+
+    def run_live_server_thread(self, connections_override=None):
+        thread = LiveServerTestCase._create_server_thread(connections_override)
+        thread.daemon = True
+        thread.start()
+        thread.is_ready.wait()
+        thread.terminate()
+        thread.join()
+
+    def test_closes_connections(self):
+        if is_connection_sqlite_in_memory(connection):
+            raise SkipTest("the sqlite backend's close() method is a no-op for "
```

Comment:
A single line is okay here as per line length guidelines at https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style.